src/constants.ts

```diff
@@ -41,6 +42,17 @@ export const monthNames = [
   'Dec',
 ];
 
+export const GOVERNANCE_OPTIONS = {
```

Comment:
What do you think about using an `enum` here?

Reply:
I also wanted to use `enum` here but then used `object` to have both id and label inside this object. If it is better, we can use `enum` here